src/client/js/partials/unsubscribe.js

```diff
@@ -0,0 +1,46 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const unsubscribeButton = document.querySelector('.js-unsubscribe-button')
+
+unsubscribeButton.addEventListener('click', async event => {
```

Comment:
This already adds the ability to unsubscribe from the general unsubscribe view so we only have to make `unsubscribeFromEmails` in `src/utils/email.js` functional again.